developer/src/kmc-package/src/kmp-compiler.ts

```diff
@@ -294,4 +297,15 @@ export default class KmpCompiler {
     // Generate kmp file
     return zip.generateAsync({type: 'binarystring', compression:'DEFLATE'});
   }
+
+  /**
+   * Legacy .kmp compiler would transform xml-format .kvk files into a binary .kvk file; now
+   * we want that to remain the responsibility of the keyboard compiler, so we'll warn the
+   * few users who are still doing this
+   */
+  private warnIfKvkFileIsNotBinary(filename: string, data: Buffer) {
+    if(filename.match(/\.kvk$/) && data.compare(Buffer.from(KvkFile.KVK_HEADER_IDENTIFIER_BYTES), 0, 3, 0, 3) != 0) {
```

Comment:
Prob not really needed, but I kinda feel this could go in a helper `isBinaryKvkFile()` that returns a boolean.

Then `warnIfKvkFileIsNotBinary` would print the warning if true.

Reply:
Will do this refactor in the future if we reuse the test elsewhere.